reports/trr0000/win/README.md

```diff
@@ -0,0 +1,249 @@
+# Pass-the-Ticket
+
+## Metadata
+
+| Key          | Value                                      |
+|--------------|--------------------------------------------|
+| ID           | TRR0000                                    |
+| External IDs | [T1550.003]                                |
+| Tactics      | Defense Evasion, Lateral Movement          |
+| Platforms    | Windows                                    |
+| Contributors | Andrew VanVleet                            |
+
+### Scope Statement
+
+There are many ways for an attacker to acquire Kerberos tickets. While this TRR
+will discuss the most common methods used in a pass-the-ticket attack, it is not
+the author's intent to make an exhaustive review of all the means by which an
+adversary could acquire a valid Kerberos ticket. This TRR will primarily focus
+on how an attacker can inject a previously-acquired ticket into their Kerberos
+ticket cache, enabling them to assume the identity of the ticket owner.
+
+## Technique Overview
+
+Kerberos is a common authentication protocol supported natively by all modern
+operating systems. It is also used extensively for authentication and
+authorization in Active Directory domains. As such, it plays a role in probably
+every modern enterprise network. An attacker who can steal valid Kerberos
+tickets from other users can use this procedure to inject those tickets into
+their own current logon session and thereby assume the identity of the ticket's
+owner.
+
+The Windows operating system has a built-in security package that implements the
+Kerberos protocol (found in the library `System32\kerberos.dll`). This security
+package supports the ability to add and retrieve tickets from a local cache as a
+fundamental part of its Kerberos implementation. Attackers can manipulate this
+functionality to acquire and insert tickets belonging to other users.
+
+## Technical Background
+
+The key concepts required to understand this attack are Kerberos tickets and the
+Windows implementation of the protocol.
+
+### Kerberos tickets
+
+Kerberos is a network authentication protocol designed to provide strong
+authentication for client/server applications by using symmetric key
+cryptography to sign 'tickets' that a client can use to prove their identity and
```

Comment:
Is there a standard or precedent for single vs double quotes? That's probably too obsessive-compulsive, but it's what my brain immediately thought. Feel free to ignore this.